src/k8s-engine/progressTracker.ts

```diff
@@ -40,6 +42,9 @@ export default class ProgressTracker {
   action<T>(description: string, priority: number, v: Promise<T> | (() => Promise<T>)) {
     const id = this.nextActionID;
 
+    if (this.backend) {
+      this.backend.lastCommandComment = description;
```

Comment:
It seems weird that we're going through an extra object to set a prop on the original object. Would having a wrapper on the backend to set the comment and call this be cleaner?

Note that in quite a few places we do things in parallel, so _which_ comment is attached is effectively random.  It will likely not match the command that failed.  So it might actually be better to just exclude this (unless you can better track the actual failure and attach the correct comment).

Reply:
I started out manually setting `this.#lastCommandComment` but soon realized that I could probably get the progress tracker, and wasn't too concerned about the parallel actions. Manually, I would set the comment before a block.

Reply:
Retracted this approach to craft the `command comments` in the lima and wsl code.